kong/plugins/rate-limiting/policies/init.lua

```diff
@@ -28,11 +28,18 @@ return {
       for period, period_date in pairs(periods) do
         if limits[period] then
           local cache_key = get_local_key(api_id, identifier, period_date, period)
-          cache.sh_add(cache_key, 0, EXPIRATIONS[period])
 
-          local _, err = cache.sh_incr(cache_key, value)
-          if err then
-            ngx_log("[rate-limiting] could not increment counter for period '"..period.."': "..tostring(err))
+          local ok, err = shm:add(cache_key, 0, EXPIRATIONS[period])
```

Comment:
I remember this issue being raised already. We could update this in this PR, or in another one :)